packages/e2e/features/ui/components/authenticator/sign-in-sms-mfa.feature

```diff
@@ -12,22 +12,15 @@ Feature: Sign In with SMS MFA
     Then I type my "phone number" with status "CONFIRMED"
     Then I type my password
     Then I click the "Sign in" button
+    Then I will be redirected to the confirm sms mfa page
     Then I see "Confirm SMS Code"
     Then I type a valid SMS confirmation code
     Then I spy request '{ "headers": { "X-Amz-Target": "AWSCognitoIdentityProviderService.RespondToAuthChallenge" } }'
     Then I click the "Confirm" button
     Then I confirm request '{"headers": { "X-Amz-Target": "AWSCognitoIdentityProviderService.RespondToAuthChallenge" } }'
 
   @angular @react @vue
-  Scenario: Sign in using a valid phone number and SMS MFA
-    When I select my country code with status "CONFIRMED"
-    Then I type my "phone number" with status "CONFIRMED"
-    Then I type my password
-    Then I click the "Sign in" button
-    Then I will be redirected to the confirm sms mfa page
-
-  @angular @react @vue
-  Scenario: Redirect to sign in page
+  Scenario: Sign in and navigate back to sign in page
```

Comment:
Renamed for clarity

Reply:
Should this test mention something about an invalid sms code?

Something like 

```suggestion
  Scenario: Sign in with sms confirmation and see invalid code error
```

Reply:
@thaddmt I think Github is just hiding the code in between this and the next chagne, this isn't actually the incorrect code error test.